cmake/par2-turbo.cmake

```diff
@@ -0,0 +1,17 @@
+set(FETCHCONTENT_QUIET FALSE)
+FetchContent_Declare(
+	par2-turbo
+	GIT_REPOSITORY  https://github.com/nzbgetcom/par2cmdline-turbo.git
+	GIT_TAG         nzbget
```

Comment:
Why specify a branch that is likely to get updated instead of a tag? Won't it break nzbget builds if breaking changes are published to the nzbget branch of par2cmdline-turbo?

Reply:
It is a tag already https://github.com/nzbgetcom/nzbget/blob/develop/cmake%2Fpar2-turbo.cmake#L31